src/pages/Loans/LoansOverview/utils/math.ts

```diff
@@ -2,6 +2,7 @@ import Big from 'big.js';
 
 import { LoanAction } from '@/types/loans';
 
+// ray test touch <<
 const calculateThresholdAmountUSD = (amountUSD: Big, threshold: Big): Big => amountUSD.mul(threshold);
 
 const calculateBorrowedAmountUSD = (
```

Comment:
These might be refactored using `calculateBorrowLimitBtcChange` and `calculateLtvAndThresholdsChange`.